generate_table.py

```diff
@@ -37,7 +37,7 @@ def generate_table(fn):
             name=name, description=other["description"], parameters=other.get("parameters", "-"),
             papers=other.get("papers", "-"), preprocessing=other.get("preprocessing", "-")
         ))
-    print("\n(generated by {} based on {})".format(__file__, os.path.basename(fn)))
+    print("\n(generated by {} based on {})".format(os.path.abspath(__file__), os.path.abspath(fn)))
```

Comment:
Sorry to nitpick, but mirroring the parameter exactly (without any normpath / abspath) is better.

Every extra step between what the computer is doing and what the user is seeing in logs, is an unnecessary obfuscation, a source of misunderstandings and errors.

Reply:
Everything is fine, I did not understand you right away, now this fixed.